# Aquatic, antibiotic-resistant <i>Aeromonas spp.</i> meta-analysis
Aeromonas 16S rRNA gene sequence dataset compilation


## Source datasets

### 1. Treated wastewater

Beattie, R. E., Skwor, T., & Hristova, K. R. (2020). Survivor microbial populations in post-chlorinated wastewater are strongly associated with untreated hospital sewage and include ceftazidime and meropenem resistant populations. [Science of The Total Environment](https://www.sciencedirect.com/science/article/pii/S0048969720337074?casa_token=iv1F7xNgfiAAAAAA:gU5u_5YeFvghDMprAboJJppcGjHLi0bVumTizm2T97Y8S42JHilexx9VlJ6_P27r4RPf_nbPoQ), 740, 140186.


### 2. Aquaculture

Bartelme, R. P., McLellan, S. L., & Newton, R. J. (2017). Freshwater recirculating aquaculture system operations drive biofilter bacterial community shifts around a stable nitrifying consortium of ammonia-oxidizing archaea and comammox Nitrospira. [Frontiers in microbiology](https://www.frontiersin.org/articles/10.3389/fmicb.2017.00101/full), 8, 101.


### 3. Rivers

McClary et al., 2021. In progress. [NCBI Sequence Read Archive](https://www.ncbi.nlm.nih.gov/bioproject/665728).



### 4. Seafood

1. (Clams) Liu, X., Teixeira, J. S., Ner, S., Ma, K. V., Petronella, N., Banerjee, S., & Ronholm, J. (2020). Exploring the Potential of the Microbiome as a Marker of the Geographic Origin of Fresh Seafood. [Frontiers in microbiology](https://www.frontiersin.org/articles/10.3389/fmicb.2020.00696/full), 11.

2. (Nase) Guivier, E., Pech, N., Chappaz, R., & Gilles, A. (2020). Microbiota associated with the skin, gills, and gut of the fish Parachondrostoma toxostoma from the Rhône basin. [Freshwater Biology](https://onlinelibrary.wiley.com/doi/abs/10.1111/fwb.13437), 65(3), 446-459.

3. (Salmon) Uren Webster, T. M., Rodriguez‐Barreto, D., Castaldo, G., Gough, P., Consuegra, S., & Garcia de Leaniz, C. (2020). Environmental plasticity and colonisation history in the Atlantic salmon microbiome: a translocation experiment. [Molecular ecology](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7078932/), 29(5), 886-898.

4. (Salmon) Webster, T. M. U., Consuegra, S., Hitchings, M., & de Leaniz, C. G. (2018). Interpopulation variation in the Atlantic salmon microbiome reflects environmental and genetic diversity. [Applied and environmental microbiology](https://aem.asm.org/content/84/16/e00691-18), 84(16).

5. (Tench) Dulski, T., Kozłowski, K., & Ciesielski, S. (2020). Habitat and seasonality shape the structure of tench (Tinca tinca L.) gut microbiome. [Scientific reports](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7064478/), 10(1), 1-11.

6. (Brown trout) Vasemägi, A., Visse, M., & Kisand, V. (2017). Effect of environmental factors and an emerging parasitic disease on gut microbiome of wild salmonid fish. [MSphere](https://msphere.asm.org/content/2/6/e00418-17), 2(6).



### 5. Beaches


Cloutier, D. D., Alm, E. W., & McLellan, S. L. (2015). Influence of land use, nutrients, and geography on microbial communities and fecal indicator abundance at Lake Michigan beaches. [Applied and environmental microbiology](https://aem.asm.org/content/81/15/4904.short), 81(15), 4904-4913.
